components/membership/registration-impl/src/test/kotlin/net/corda/membership/impl/registration/RegistrationProxyImplTest.kt

```diff
@@ -283,21 +280,26 @@ class RegistrationProxyImplTest {
             registrationId: UUID,
             member: HoldingIdentity,
             context: Map<String, String>
-        ): MembershipRequestRegistrationResult = registrationResult1
+        ) {
+
+        }
```

Comment:
```suggestion
        ) = Unit
```
Maybe something like this would be cleaner than empty function bodies?